config/consumer.properties

```diff
@@ -17,13 +17,13 @@
 # Zookeeper connection string
 # comma separated host:port pairs, each corresponding to a zk
 # server. e.g. "127.0.0.1:3000,127.0.0.1:3001,127.0.0.1:3002"
-zookeeper.connect=127.0.0.1:2181
+#zookeeper.connect=127.0.0.1:2181
 
 # timeout in ms for connecting to zookeeper
-zookeeper.connection.timeout.ms=6000
+#zookeeper.connection.timeout.ms=6000
 
 #consumer group id
-group.id=test-consumer-group
+#group.id=test-consumer-group
 
 #consumer timeout
-#consumer.timeout.ms=5000
+consumer.timeout.ms=3000
```

Comment:
Ah, my bad. I made these change only to test the patch. The change was reverted just now.

Thanks @ijuma.